admin/src/main/scala/io/buoyant/admin/PrometheusStatsHandler.scala

```diff
@@ -0,0 +1,52 @@
+package io.buoyant.admin
+
+import com.twitter.finagle.Service
+import com.twitter.finagle.http.{MediaType, Request, Response}
+import com.twitter.finagle.stats.MetricsStatsReceiver
+import com.twitter.io.Charsets
+import com.twitter.util.Future
+import org.jboss.netty.buffer.ChannelBuffers
+
+/**
+ * PrometheusStatsHandler
+ *
+ * Export Stats in a Prometheus-compatible format. Prometheus (http://prometheus.io/)
+ * exports stats in a somewhat different format than the Finagle's MetricsExporter. Specifically,
+ * instead of delimiting scopes by forward slashes, a colon is used. For stats, instead of exporting
+ * percentiles (p50, p99, etc.) and other quantities (max, min, etc.) as individual stats, they
+ * should be exported as labels in the Prometheus format.
+ */
+class PrometheusStatsHandler extends Service[Request, Response] {
+  private[this] lazy val registry = MetricsStatsReceiver.defaultRegistry
```

Comment:
i'd take a registry in the constructor so this can be tested.  Something like:
```scala
class PrometheusStatsHandler(registry: MetricsRegistry) {
  ...
}
```